Assets/SEE/Game/SceneManipulation/GameNodeMarker.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using SEE.DataModel.DG;
+using SEE.Game.City;
+using SEE.GO;
+using UnityEngine;
+
+namespace SEE.Game.SceneManipulation
+{
+
+    public static class GameNodeMarker
+    {
+        public static GameObject NodeMarker(GameObject parent, Vector3 position, Vector3 worldSpaceScale)
+        {
+            GameObject sphere = GameObject.CreatePrimitive(PrimitiveType.Sphere);
+            Vector3 position = parent.transofrm.position;

```

Comment:
Typo in `transofrm` leading to a compile error. Do not check-in code that does not compile!